Library/Homebrew/cmd/shellenv.sh

```diff
@@ -5,31 +5,32 @@
 #:  The variables `HOMEBREW_PREFIX`, `HOMEBREW_CELLAR` and `HOMEBREW_REPOSITORY` are also exported to avoid querying them multiple times.
 #:  Consider adding evaluation of this command's output to your dotfiles (e.g. `~/.profile`, `~/.bash_profile`, or `~/.zprofile`) with: `eval $(brew shellenv)`
 
+# shellcheck disable=SC2248,SC2154
```

Comment:
Add a comment above explaining this

Reply:
add `disable=SC2248` to avoid showing the message below.
```
In /usr/local/Homebrew/Library/Homebrew/cmd/shellenv.sh line 11:
  case "$(/bin/ps -p ${PPID} -c -o comm=)" in
                     ^-----^ SC2248: Prefer double quoting even when variables don't contain special characters.
```